src/modules/sdlog2/sdlog2.c

```diff
@@ -103,6 +104,8 @@
 #include "sdlog2_format.h"
 #include "sdlog2_messages.h"
 
+#define PX4_EPOCH_SECS 1234567890ULL
```

Comment:
Individual macros are already bad, but I did so in anticipation of the architecture change inducing a px4.h for all processes. I've logged this in an issue so we don't forget.
https://github.com/PX4/Firmware/issues/1579